scripts/main.js

```diff
@@ -1,5 +1,50 @@
 var helloCork = function() {
-  console.log('Hello Cork');
+  	console.log('Hello Cork');
 };
 
+var signedUrlUpcoming = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&sig=d07ac4460173bc86aa05e0c5b65acd79b2b8ae1b";
+var signedUrlPast = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&status=past&sig=052868f6f4d1f5a525fa6ca28a74e771165ad4ea";
+
+var getUpcomingEvents = function() {
+	var call = $.ajax({
+		type: "GET",
+		dataType: "jsonp",
```

Comment:
I used jsonp because I didn't know how to fix this:

![corserror](https://cloud.githubusercontent.com/assets/15719653/22183574/6b9ea9ba-e0b9-11e6-935b-54325ab2cb5a.PNG)

Is jsonp just json designed to be used for Asynchronous calls?